sdk/storage/storage-blob/package.json

```diff
@@ -132,13 +133,19 @@
   },
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/core-http": "^3.0.0",
+    "@azure/core-auth": "^1.4.0",
+    "@azure/core-client": "^1.6.2",
+    "@azure/core-http-compat": "^2.0.0",
     "@azure/core-lro": "^2.2.0",
     "@azure/core-paging": "^1.1.1",
+    "@azure/core-rest-pipeline": "^1.10.1",
     "@azure/core-tracing": "1.0.0-preview.13",
```

Comment:
@xirzec: Do you think `core-tracing` could/should be upgraded to `^1.0.0` as part of this PR, or is this a separate batch of work?  I noticed the only packages still depending on `core-tracing@1.0.0-preview.13` are storage and core-http, so I was wondering if removing `core-http` might allow us to easily upgrade `core-tracing`.

Reply:
I started down the path of doing it in this PR but decided it would be better off being a follow-up PR since the changes are largely mechanical and would be easier to review that way. We could bring it into this PR but I do think it would create more noise in an already large change.

Reply:
We also have to migrate the test recorder in a follow-up PR as well.

Reply:
If additional code changes are required to update `core-tracing` I agree a separate PR is better.